### ISSUE TITLE
social network feed

### ISSUE THREAD

**Issue by 9px** [fguillot/picoFeed#219]
hi. thanks for your great works.
is it possible to get social account (such as twitter) with picoFeed?

**Comment by BernhardPosselt** [comment]
Picofeed used to handle Twitter just fine when they provided an RSS feed. Then they locked down their api and made it very hard to use: you need to get a token for your library and one for every Twitter user that accesses the api. The website for getting both tokens is ridiculously hard to use so asking your users to get a token is out of question. To make matters worse both tokens are limited to a small amount of queries per day.

If you ask me, all social networks and especially Twitter try to keep their information locked in to keep views and interaction on their page as much as possible (ads!), that's why they are all rolling out very hard to use apis.

Those APIs also work fundamentally different than RSS/atom feeds so you're better off using a dedicated library, see https://dev.twitter.com/overview/api/twitter-libraries

**Comment by BernhardPosselt** [comment]
Tldr: social media APIs make it hard to impossible to use their feed like content in a feed reader properly.
